week2-section2/exercise2/PlaylistContainer.cpp

```diff
@@ -28,10 +23,11 @@
 		}
 	};
 
-	FPlaylist FPlaylistContainer::GetPlaylist ( int InPlaylistIndex) const
+	//------------------------------------------------------------
+	FPlaylist FPlaylistContainer::GetPlaylist(int InPlaylistIndex) const
 	{
 		std::cout << "\nResults: " << std::endl;
-		if (Playlists[InPlaylistIndex].GetPlaylistTitle() != "")
+		if (Playlists[InPlaylistIndex].IsPlaylistValid()  == true)
```

Comment:
Remember that the `== true` is not necessary. You can simply use the result of the method itself since it returns a boolean.